cmd/edge-proxy/main.go

```diff
@@ -190,6 +191,7 @@ func main() {
 		}
 	}(certificate)
 
+	https.StartHTTPSProxy()
```

Comment:
It might be better to put https handler under `server` module as well. Just like other proxy servers.

Reply:
done